contracts/Strategy.sol

```diff
@@ -350,6 +348,12 @@ contract Strategy is BaseStrategy {
         lpStaker.deposit(liquidityPoolIDInLPStaking, _amountToStake);
     }
 
+    function unstakeLP(uint256 amountToUnstake) external onlyVaultManagers {
+        if (amountToUnstake > 0 && balanceOfStakedLPToken() > 0) {
+            _unstakeLP(Math.min(amountToUnstake, balanceOfStakedLPToken()));
```

Comment:
For consistency, can Math.min either be input into the _unstakeLP function or it be pulled out of _withdrawFromLP into withdrawFromLP?